perllib/FixMyStreet/App/Controller/Around.pm

```diff
@@ -398,10 +398,13 @@ sub _geocode : Private {
 sub lookup_by_ref : Private {
     my ( $self, $c, $ref ) = @_;
 
-    my $problems = $c->cobrand->problems->search([
-        id => $ref,
-        external_id => $ref
-    ]);
+    my $problems;
+    unless ( $problems = $c->cobrand->call_hook("lookup_by_ref", $ref) ) {
```

Comment:
Unless we foresee some weird things being done by this hook, should it perhaps return the parameter for `search()` but not do the search itself? Then it could return nothing to stick with the default also.